<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1664005317954' stop='1664005377971'>
    <name>DeActivatecash</name>
    <title>DeActivatecash</title>
    <test-cases>
        <test-case start='1664005317954' status='broken' stop='1664005377970'>
            <name>DeActivate cash register</name>
            <title>DeActivate cash register</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-105.0.5195.127'/>
            </parameters>
            <steps>
                <step start='1664005317954' status='broken' stop='1664005377970'>
                    <name>POST /session/:sessionId/url</name>
                    <title>POST /session/:sessionId/url</title>
                    <attachments>
                        <attachment title='Request' source='42f3d3d6-f393-4769-bf57-1d9e0918be5d-attachment.json' type='application/json' size='48'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>Timeout of 60000ms exceeded. The execution in the test "DeActivatecash DeActivate cash register" took too long. Try to reduce the run time or increase your timeout for test specs (https://webdriver.io/docs/timeouts). (C:\Users\LENOVO\Desktop\Wdio_Scripts\test\specs\Setmore_cash.js)</message>
                <stack-trace>Error: Timeout of 60000ms exceeded. The execution in the test "DeActivatecash DeActivate cash register" took too long. Try to reduce the run time or increase your timeout for test specs (https://webdriver.io/docs/timeouts). (C:\Users\LENOVO\Desktop\Wdio_Scripts\test\specs\Setmore_cash.js)
                    at createTimeoutError (C:\Users\LENOVO\Desktop\Wdio_Scripts\node_modules\mocha\lib\errors.js:498:15)
                    at Test.Runnable._timeoutError (C:\Users\LENOVO\Desktop\Wdio_Scripts\node_modules\mocha\lib\runnable.js:431:10)
                    at Timeout.&lt;anonymous> (C:\Users\LENOVO\Desktop\Wdio_Scripts\node_modules\mocha\lib\runnable.js:246:24)
                    at listOnTimeout (node:internal/timers:559:17)
                    at processTimers (node:internal/timers:502:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>